website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md

```diff
@@ -11,7 +11,7 @@ excerpt: How are TikTok, Flipkart, KubeSphere, and others using OpenEBS for Loca
 
 We have recently seen a massive increase in the usage of different flavors of OpenEBS Local PV. We estimate by looking at container pulls for underlying components combined with some call home data for those users of OpenEBS that enable the capturing of metrics that the weekly new deployments of OpenEBS for LocalPV increased by nearly 10x during 2020. This can be attributed to the fact that more and more cloud native Stateful applications are moving into Kubernetes
 
-![LocalPv Deployment](https://admin.mayadata.io/content/images/2021/03/Local-PV-Deployment.PNG)
+![LocalPv Deployment](/images/blog/Local-PV-Deployment.png)
```

Comment:
Is this all @rakeshPRaghu  ? Are there no more images that are missing ?

Reply:
Yes @satyapriyamishra222, I have gone through the blogs again only these images were missing.